Kickstarter-iOS/Views/Cells/DeprecatedRewardCell.swift

```diff
@@ -11,42 +11,42 @@ internal final class DeprecatedRewardCell: UITableViewCell, ValueCell {
-  @IBOutlet fileprivate weak var rewardTitleLabel: UILabel!
-  @IBOutlet fileprivate weak var rootStackView: UIStackView!
-  @IBOutlet fileprivate weak var selectRewardButton: UIButton!
-  @IBOutlet fileprivate weak var shippingLocationsLabel: UILabel!
-  @IBOutlet fileprivate weak var shippingLocationsStackView: UIStackView!
-  @IBOutlet fileprivate weak var shippingLocationsSummaryLabel: UILabel!
+  @IBOutlet fileprivate var allGoneContainerView: UIView!
+  @IBOutlet fileprivate var allGoneLabel: UILabel!
+  @IBOutlet fileprivate var cardView: UIView!
+  @IBOutlet fileprivate var checkmarkImageView: UIImageView!
+  @IBOutlet fileprivate var conversionLabel: UILabel!
+  @IBOutlet fileprivate var descriptionLabel: UILabel!
+  @IBOutlet fileprivate var estimatedDeliveryDateLabel: UILabel!
+  @IBOutlet fileprivate var estimatedDeliveryLabel: UILabel!
+  @IBOutlet fileprivate var estimatedDeliveryDateStackView: UIStackView!
+  @IBOutlet fileprivate var footerLabel: UILabel!
+  @IBOutlet fileprivate var footerStackView: UIStackView!
+  @IBOutlet fileprivate var includesTitleLabel: UILabel!
+  @IBOutlet fileprivate var itemsContainerStackView: UIStackView!
+  @IBOutlet fileprivate var itemsHeaderStackView: UIStackView!
+  @IBOutlet fileprivate var itemsStackView: UIStackView!
+  @IBOutlet fileprivate var manageRewardButton: UIButton!
+  @IBOutlet fileprivate var minimumLabel: UILabel!
+  @IBOutlet fileprivate var minimumStackView: UIStackView!
+  @IBOutlet fileprivate var rewardTitleLabel: UILabel!
+  @IBOutlet fileprivate var rootStackView: UIStackView!
+  @IBOutlet fileprivate var selectRewardButton: UIButton!
+  @IBOutlet fileprivate var shippingLocationsLabel: UILabel!
+  @IBOutlet fileprivate var shippingLocationsStackView: UIStackView!
+  @IBOutlet fileprivate var shippingLocationsSummaryLabel: UILabel!
   @IBOutlet fileprivate var separatorViews: [UIView]!
-  @IBOutlet fileprivate weak var titleDescriptionStackView: UIStackView!
-  @IBOutlet fileprivate weak var viewYourPledgeButton: UIButton!
-  @IBOutlet fileprivate weak var youreABackerCheckmarkImageView: UIImageView!
-  @IBOutlet fileprivate weak var youreABackerContainerView: UIView!
-  @IBOutlet fileprivate weak var youreABackerLabel: UILabel!
-  @IBOutlet fileprivate weak var youreABackerStackView: UIStackView!
+  @IBOutlet fileprivate var titleDescriptionStackView: UIStackView!
+  @IBOutlet fileprivate var viewYourPledgeButton: UIButton!
+  @IBOutlet fileprivate var youreABackerCheckmarkImageView: UIImageView!
+  @IBOutlet fileprivate var youreABackerContainerView: UIView!
+  @IBOutlet fileprivate var youreABackerLabel: UILabel!
+  @IBOutlet fileprivate var youreABackerStackView: UIStackView!
 
   internal override func awakeFromNib() {
     super.awakeFromNib()
 
-    let tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(tapped))
+    let tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(DeprecatedRewardCell.tapped))
```

Comment:
Since `SwiftFormat` would correct this to `#selector(self.tapped)` I explicitly provided the class name since I'm not entirely sure or haven't seen the use of `self` in a selector before.